styles/blocks/footer/footer.less

```
@@ -0,0 +1,101 @@
+@social: facebook, twitter, vk, inst;
+@app: apple, google;
+
+.footer {
+  color: @text-grey;
+  background-color: @black;
+
+  &__wrapper {
+    display: flex;
+    flex-direction: column;
+    gap: 15px;
+    padding: 30px 0 22px;
+
+    @media @screen-gt-s {
+      padding-bottom: 30px;
+      gap: 30px;
+    }
+
+    @media @screen-gt-m {
+      padding-top: 40px;
+      padding-bottom: 35px;
+    }
+  }
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    gap: 30px;
+    border-bottom: 1px solid @border-grey;
+    padding-bottom: 30px;
+
+    @media @screen-gt-s {
+      flex-direction: row;
+      justify-content: space-between;
+    }
+
+    @media @screen-gt-m {
+      align-items: center;
+      padding-bottom: 40px;
+    }
+  }
+
+  &__social {
+    .list;
```

```suggestion
    .list();
```
без скобок тоже работает, но deprecated https://lesscss.org/features/#mixins-feature

---

принял, поправлю👍